openquake/server/db/actions.py

```diff
@@ -268,31 +268,30 @@ def del_calc(db, job_id, user):
     dependent = db(
         'SELECT id FROM job WHERE hazard_calculation_id=?x', job_id)
     if dependent:
-        return ('Cannot delete calculation %d: there are calculations '
-                'dependent from it: %s' % (job_id, [j.id for j in dependent]))
+        return {"error": 'Cannot delete calculation %d: there '
+                'are calculations '
+                'dependent from it: %s' % (job_id, [j.id for j in dependent])}
     try:
         owner, path = db('SELECT user_name, ds_calc_dir FROM job WHERE id=?x',
                          job_id, one=True)
     except NotFound:
-        return ('Cannot delete calculation %d: ID does not exist' % job_id)
+        return {"error": 'Cannot delete calculation %d:'
+                ' ID does not exist' % job_id}
 
     deleted = db('DELETE FROM job WHERE id=?x AND user_name=?x',
                  job_id, user).rowcount
     if not deleted:
-        return ('Cannot delete calculation %d: it belongs to '
-                '%s and you are %s' % (job_id, owner, user))
+        return {"error": 'Cannot delete calculation %d: it belongs to '
+                '%s and you are %s' % (job_id, owner, user)}
 
-    # try to delete datastore and associated files
+    # try to delete datastore and associated file
     # path has typically the form /home/user/oqdata/calc_XXX
-    fnames = []
-    for fname in glob.glob(path + '.*'):
-        try:
-            os.remove(fname)
-        except OSError as exc:  # permission error
-            print('Could not remove %s: %s' % (fname, exc))
-        else:
-            fnames.append(fname)
-    return fnames
+    fname = path + ".hdf5"
+    try:
+        os.remove(fname)
+    except OSError as exc:  # permission error
+        {"error": 'Could not remove %s: %s' % (fname, exc)}
```

Comment:
Forgot it, forgive me.
Added here https://github.com/gem/oq-engine/pull/2905/commits/df1a92af8f8acfce0d5d94b8f40b06a279899ec2